src/main/kotlin/org/rust/lang/core/psi/RsPsiFactory.kt

```diff
@@ -471,6 +471,9 @@ class RsPsiFactory(
         else -> createExpressionOfType("${expr.text}.$methodNameText()")
     }
 
+    fun createNotEqZero(lhs: RsExpr): RsBinaryExpr =
```

Comment:
I think that this is may be too specific to put into `RsPsiFactory`. You can just call `createExpression` and cast the result to `RsBinaryExpr`.